tools/cockpit.spec

```diff
@@ -155,15 +155,21 @@ BuildRequires:  python3-pytest-timeout
 %check
 make -j$(nproc) check
 
-%if 0%{?rhel} == 0
+%if 0%{?rhel} == 0 && 0%{?suse_version} == 0
 export NO_QUNIT=1
 %pytest
 %endif
 
 %install
 %make_install
+
+%if 0%{?suse_version} > 1500
+mkdir -p $RPM_BUILD_ROOT%{_pam_vendordir}
+install -p -m 644 tools/cockpit.pam $RPM_BUILD_ROOT%{_pam_vendordir}/cockpit
+%else
```

Comment:
Alternatively we create a new spec file variable for the cockpit_pam_dir. Then the install line stays the same.